specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -4346,13 +4346,27 @@
       "description": "settings for security policy patterns to match",
       "type": "object",
       "properties": {
+        "IsProfileLevel": {
+          "description": "Flag to signal whether the WAF policy association is at the default Profile level.",
+          "enum": [
+            "True",
+            "False"
+          ]
+        },
```

Comment:
I don't think we need to change any existing api-versions. We will need new stable API versions or update it in preview API versions. Otherwise, we cannot pass the review from ARM API team.